x-pack/plugins/apm/server/lib/transactions/charts/get_anomaly_data/index.ts

```diff
@@ -55,6 +66,40 @@ export async function getAnomalySeries({
     return;
   }
 
-  // TODO [APM ML] return a series of anomaly scores, upper & lower bounds for the given timeSeriesDates
-  return;
+  const mlJobIds = await getMLJobIds(setup.ml, environment);
+
+  // don't fetch anomalies if there are more than 1 ML jobs for the given environment
+  if (mlJobIds.length > 1) {
+    return;
+  }
+  const jobId = mlJobIds[0];
+
+  const mlBucketSize = await getMlBucketSize({
+    serviceName,
+    transactionType,
+    setup,
+    jobId,
+  });
+
+  const { start, end } = setup;
+  const { intervalString, bucketSize } = getBucketSize(start, end, 'auto');
+
+  const esResponse = await anomalySeriesFetcher({
+    serviceName,
+    transactionType,
+    intervalString,
+    mlBucketSize,
+    setup,
+    jobId,
+  });
+
+  return esResponse
+    ? anomalySeriesTransform(
+        esResponse,
+        mlBucketSize,
+        bucketSize,
+        timeSeriesDates,
+        jobId
+      )
+    : undefined;
```

Comment:
An `if` is simpler than a ternary in this case:

```suggestion
  if(esResponse && mlBucketSize) {
    return anomalySeriesTransform(
        esResponse,
        mlBucketSize,
        bucketSize,
        timeSeriesDates,
        jobId
      )
  }
```